### PR TITLE
Fixed Paths

### PR DESCRIPTION
**Pull request by kiquance21**

### CODE REVIEW

File: software/bringup-scripts/pic_hdmi.sh [hunk 1]
```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 
-./../power_init.sh
+./../scripts/power_init.sh
```
Comment (rroohhh) [inline software/bringup-scripts/pic_hdmi.sh]: the `./` part is not needed, only `../scripts/power_init.sh` is sufficient

(this applies to all the changes not just this one)

File: software/scripts/i2c0.func [hunk 2]
```diff
@@ -1,7 +1,6 @@
 #!/bin/sh
 
-. ./bit.func
-
+. ../scripts/bit.func
```
Comment (rroohhh) [inline software/scripts/i2c0.func]: unnecessary change

File: software/scripts/pass_init.sh [hunk 3]
```diff
@@ -11,5 +11,5 @@ for n in `seq 0 63`; do
     echo 0 >$gpio/value
 done
 
-./rf_disable.sh
+./../bringup-scripts/rf_disable.sh
```
Comment (rroohhh) [inline software/scripts/pass_init.sh]: `pass_init.sh` should also be moved to the bringup-scripts

Comment (kiquance21) [reply software/scripts/pass_init.sh]: Ok,I'll get it done in some while

File: software/scripts/i2c0.func (hunk 2 above)
Comment (kiquance21) [reply software/scripts/i2c0.func]: This was required by `prep_icsp.sh` .It includes `i2c0.func` which further includes `bit.func` ,so if the  `prep_icsp.sh`  is run , it can't find `bit.func`.I would have liked to do something more generic but couldn't find anything

Comment (rroohhh) [reply software/scripts/i2c0.func]: I see, for that you can use something like `cd $(dirname $(realpath $0))`

**Comment by rroohhh** [comment]
Also can I close #97 as we are continuing here?

File: software/scripts/i2c0.func (hunk 2 above)
Comment (kiquance21) [reply software/scripts/i2c0.func]: I couldn''t find a proper way to do that ,once way was to use `cd $(dirname $BASH_SOURCE)` in the `i2c0.func` file

Comment (rroohhh) [reply software/scripts/i2c0.func]: what's the problem with the line i suggested?

Comment (kiquance21) [reply software/scripts/i2c0.func]: `cd $(dirname $(realpath $0))` lets us switch to the path of the source script (`bringup-scripts`),but does not help in finding the path of `bit.func`

Comment (rroohhh) [reply software/scripts/i2c0.func]: is does, when putting `cd $(dirname $(realpath $0))` inside `i2c0.func`

Comment (kiquance21) [reply software/scripts/i2c0.func]: Yes this would work ,but we would have to` cd `back to the `bringup-scripts` folder after sourcing the `i2c0.func` file.
If we are going with this should I change `shebang ` to `bash` for all the scripts or only one which are required

Comment (rroohhh) [reply software/scripts/i2c0.func]: yes, of course, `pushd` and `popd` should work fine for that

File: software/bringup-scripts/prep_icsp.sh [hunk 4]
```diff
@@ -5,9 +5,8 @@
 ../scripts/fclk_init.sh
 
 ../scripts/gpio.py init
-
 . ../scripts/i2c0.func
-
+popd
```
Comment (rroohhh) [inline software/bringup-scripts/prep_icsp.sh]: Is there a reason for this `popd`?

Comment (kiquance21) [reply software/bringup-scripts/prep_icsp.sh]: Sorry,must be a typo

**Comment by rroohhh** [comment]
superseded by fb4149086a4d8815bc7eb4d44a96a50c24b735c0
